app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -330,26 +330,30 @@ function initializePodcastPlayback() {
     pausePodcastBar();
   }
 
-  function playPause(audio) {
+  function ahoyMessage(action) {
```

Comment:
These changes keep the same logic but restructures a bit for clarity. It also follows with the "switch" from the video logic below